src/main.js

```diff
@@ -3,12 +3,32 @@
 const API_KEY = 'e9979803c3894fa182b399c482279297';
 const HOST_URL = 'https://newsapi.org/v2';
```

Comment:
Better to end link with '/' to avoid extra redirects.
Also do not forget to remove extra '/' from `SOURCES_PARAM ` variable.

Reply:
Fixed